tests/unit/amo/components/TestContributeCard.js

```diff
@@ -48,7 +48,7 @@ describe(__filename, () => {
   });
 
   it('does not render anything if add-on has no contributions URL', () => {
-    const root = render({ addon: createAddon({ contributions_url: null }) });
+    const root = render({ addon: createAddon({ contributions_url: '' }) });
```

Comment:
Could you also update `fakeAddon` to have `contributions_url: ''` ?

https://github.com/mozilla/addons-frontend/blob/bba92d7564362ef303d358bc7e59e8422ef81ba5/tests/unit/helpers.js#L101